google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/spi/v1/MessageDispatcher.java

```diff
@@ -273,7 +273,8 @@ public void processReceivedMessages(List<com.google.pubsub.v1.ReceivedMessage> r
       ackHandlers.add(new AckHandler(pubsubMessage.getAckId(), messageSize));
     }
     Instant expiration = now.plus(messageDeadlineSeconds * 1000);
-    logger.log(Level.INFO, "Received " + responseMessages.size() + " messages at " + now);
+    logger.log(
+        Level.FINER, "Received {0} messages at {1}", new Object[] {responseMessages.size(), now});
```

Comment:
OK about using FINE, but I don't think it's possible to use varargs. The signature of [log](https://docs.oracle.com/javase/7/docs/api/java/util/logging/Logger.html#log(java.util.logging.Level,%20java.lang.String,%20java.lang.Object[])) is either one object or array of objects, not varargs.